ctapipe/reco/hillas_intersection.py

```diff
@@ -168,7 +168,7 @@ def predict(self, hillas_dict, subarray, array_pointing, telescopes_pointings=No
         err_x *= u.rad
         err_y *= u.rad
 
-        nom = SkyCoord(fov_lon=src_x * u.rad, fov_lat=src_y * u.rad, frame=nom_frame)
+        nom = SkyCoord(fov_lat=src_x * u.rad, fov_lon=src_y * u.rad, frame=nom_frame)
```

Comment:
This bug would proablly also not have happened, if the variables for called `src_fov_lat` / `src_fov_lon` instead of `x / y` / properly use the nominal / telescope frame.

Reply:
Or the bug would just move to the other code point.

So all in all I would say this is another reason to just fully switch to hillas parameters in telescope frame with names field names mapping to the coordinate frame axis names.

Reply:
Agreed, of course when I wrote this code the nominal frame dimensions were called x and y :-).

Reply:
> So all in all I would say this is another reason to just fully switch to hillas parameters in telescope frame with names field names mapping to the coordinate frame axis names.

I completely agree with this statement. The definition of Hillas parameters in the camera frame has always upset me (for many reasons). In HESS we typically define them in the nominal system, however we never divergently point, so the difference to the telescope frame is minimal.

Reply:
fortunately, we have #1591  and #1408  coming soon...